### ISSUE TITLE
Error When attempting to install and load plugin

### ISSUE THREAD

**Issue by Nitrousoxide** [Critical-Impact/InventoryTools#269]
**Describe the bug**
Attempting to install the plugin gives an error "Failed to load"  Restarting the game and retrying does not fix it.  Other plugins have successfully been installed and run.  

**To Reproduce**
1: Open up the plugin list
2: Select to install plugin
3. See error

**Expected behavior**
Plugin should load

**Screenshots**
![image](https://github.com/user-attachments/assets/64810ee7-a62d-4f95-b4c3-c4674186b478)

**Dalamud Information:**
[dalamud.log](https://github.com/user-attachments/files/16480691/dalamud.log)

**Comment by Critical-Impact** [comment]
This error appears to be related to the plugin being unable to load parts of the games data.
Do you have any textools mods, exd replacements or modifications to your games files?
I'd recommend running a game repair with xivlauncher, if you are are using textools or fan translations that modify the games data I can't really help you there

**Comment by Nitrousoxide** [comment]
No mods or anything.  I just went and removed any plugins in case they were conflicting and... ran a repair:
![image](https://github.com/user-attachments/assets/b4684f94-453e-454c-b9d7-0aea3e5ffbdb)

It said it repaired 7 game files.

I was now able to install Allagan Tools (and the other plugins I was using too)

![image](https://github.com/user-attachments/assets/924e35eb-c4b9-472d-98c9-71165316eb09)

So I guess I got some corrupted game files or something at some point.

I'm gonna close this issue since it's resolved for me.
